sdks/go/alpha.go

```diff
@@ -98,57 +99,57 @@ func (a *Alpha) GetCounterCount(key string) (int64, error) {
 // IncrementCounter increases a counter by the given nonnegative integer amount.
 // Will execute the increment operation against the current CRD value. Will max at max(int64).
 // Will error if the key was not predefined in the GameServer resource on creation.
-// Returns false if the count is at the current capacity (to the latest knowledge of the SDK),
+// Returns error if the count is at the current capacity (to the latest knowledge of the SDK),
 // and no increment will occur.
 //
 // Note: A potential race condition here is that if count values are set from both the SDK and
 // through the K8s API (Allocation or otherwise), since the SDK append operation back to the CRD
 // value is batched asynchronous any value incremented past the capacity will be silently truncated.
-func (a *Alpha) IncrementCounter(key string, amount int64) (bool, error) {
+func (a *Alpha) IncrementCounter(key string, amount int64) error {
 	if amount < 0 {
-		return false, errors.Errorf("CountIncrement amount must be a positive int64, found %d", amount)
+		return errors.Errorf("CountIncrement amount must be a positive int64, found %d", amount)
 	}
 	_, err := a.client.UpdateCounter(context.Background(), &alpha.UpdateCounterRequest{
 		CounterUpdateRequest: &alpha.CounterUpdateRequest{
 			Name:      key,
 			CountDiff: amount,
 		}})
 	if err != nil {
-		return false, errors.Wrapf(err, "could not increment Counter %s by amount %d", key, amount)
+		return errors.Wrapf(err, "Could not increment Counter %s by amount %d", key, amount)
```

Comment:
As a rule, error message should not be capitalised - see: https://google.github.io/styleguide/go/decisions.html#error-strings

(This should be changed on all error messages)